angular/build/config/vendor.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Vendor is hand written; has to be, because we have no way of automatically
knowing what files from within the node_modules to include